apps/designer/app/designer/features/props-panel/control.stories.tsx

```diff
@@ -7,22 +7,14 @@ export default {
 
 export const Primitive: ComponentStoryObj<typeof Control> = {
   args: {
-    type: "text",
+    component: "Image",
+    userProp: { prop: "src", id: "some_id", value: "" },
```

Comment:
we never use snake case, only in CONSTANTS_LIKE stuff, lets not even make an example lol to prevent this idea